CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+# Contributor Guide
+
+## Open Discussion
+
+The most important part of an open source project is *open discussion*. If a developer, user, or other contributor has
+a question, comment, suggestion, or concern about any part of this project, **please speak up**: create an issue or add
+a comment to an existing issue or pull request.
```

Comment:
> create chat room 

A few people here and here have been in https://jabbr.net/#/rooms/CodeContracts but most of the action is happening here in github issues.